Atomic/backends/_containers_storage.py

```diff
@@ -53,7 +60,26 @@ def pull_image(self, image, remote_image_obj, **kwargs):
         :param pull_args:
         :return:
         """
-        raise UnderDevelopment()
+        debug = kwargs.get('debug', False)
+        fq_name = remote_image_obj.fq_name
+        registry, _, _, tag, _ = util.Decompose(fq_name).all
+        if not image.endswith(tag):
+            image += ":{}".format(tag)
+        if '@sha256:' in image:
+            image = image.replace("@sha256:", ":")
+
+        insecure = False
+        registries_config = util.load_registries_from_yaml()
+        if "insecure_registries" in registries_config:
+            if registry in registries_config['insecure_registries']:
+                insecure = True
+        source = "docker://{}".format(image)
```

Comment:
will the source/dest always be hardcoded?  Can/should we have them as parameters, perhaps optional, to this function?

Reply:
Yes they are for now.  We haven't talked about opening up the transports yet.  Each other backend is more or less this way.  Certainly something that we can discuss in the future but the assumption was fancy stuff be done with skopeo or the individual tools

Reply:
Could check to see if the transport is specified?
atomic pull --storage container-storage dir:/tmp/alpine.tar 

But I agree we should move this to a future version.